test/assertion.js

```diff
@@ -336,7 +336,7 @@ describe('Assertion', function(){
         .error('cannot POST /json/identify?baz=foo (400)', done);
     });
 
-    it('should throw if error messages dont match', function(done){
+    it('should throw if error messages does not match', function(done){
```

Comment:
singular `messages` if you're actually concerned with "engrish" here. (there's probably a better commit message for this.)